code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -133,6 +133,12 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 /datum/gas_mixture/proc/return_volume()
 	return max(0, volume)
 
+/// Gets the gas visuals for everything in this mixture
+/datum/gas_mixture/proc/return_visuals()
+	var/list/output
+	GAS_OVERLAYS(gases, output)
+	return output
```

Comment:
Are you sure we should be standardizing this? gas overlays scale in a manner that's meant to make standard flow (not excited groups) look better, should we really be using it for this too?

Also don't you want to use a different molar count threshold for this then turfs do? for similar reasons.

Reply:
If we later want to make it scale based on pressure or some such we can put that code here, for now though I'm not too bothered by a plain molar count being used.

Reply:
and the threshold? shouldn't this be something a bit larger, since gas isn't really liable to spread when it's in a canister?

Reply:
The window is small so the stronger visual effects are preferred anyway to make it more visible.

Reply:
aight